src/main/java/mate/academy/bookstore/controller/OrderController.java

```diff
@@ -0,0 +1,77 @@
+package mate.academy.bookstore.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.OrderItemResponseDto;
+import mate.academy.bookstore.dto.OrderRequestDto;
+import mate.academy.bookstore.dto.OrderResponseDto;
+import mate.academy.bookstore.dto.OrderUpdateStatusRequestDto;
+import mate.academy.bookstore.model.User;
+import mate.academy.bookstore.service.OrderService;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order management", description = "Endpoints for managing orders")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/orders")
+public class OrderController {
+    private final OrderService orderService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "Create new order")
```

Comment:
Please, add the description everywhere
```suggestion
    @Operation(summary = "Create new order",
                         description = "message")
```